packages/x-components/static-docs/experience-search-and-discovery/README.md

```diff
@@ -130,3 +103,29 @@ your native app search solution.
     'x_empathize',
     'x_results',]"
 />
+
+::: warning
+
+Some X&nbsp;Components are supported by Empathy Platform search and discovery features. If you’re
+using a different search engine, make sure it supports all the functionalities you need.
+
+:::
+
+## Try X Components to…
+
+- Surprise your shoppers by **rendering shopping-aimed search and discovery features** that
+  intuitively guide and inspire them with trends, suggestions, and recommendations, based on your
+  shoppers’ behavior.
+- Add a **predictive layer** to your search UI that gathers all search functionalities in one, from
+  autocomplete and autosuggest to ideas of what to look for next.
```

Comment:
```suggestion
- Add a **predictive layer** to your search UI, gathering all search functionalities in one: from
  autocomplete and autosuggest to ideas of what to look for next.
```

Reply:
It's a common -ING mistake to use a gerund form to express a sequential action, rather than a simultaneous action. Here you can't gather all search functionalities at the same time you are adding the predictive layer. You got all the search functionalities in one as a consequence of implementing the predictive layer. So, it's better to avoid using -ING form here :)